libs/user-facing-errors/src/query_engine.rs

```diff
@@ -312,3 +312,15 @@ pub struct MissingFullTextSearchIndex {}
     message = "Prisma needs to perform transactions, which requires your MongoDB server to be run as a replica set. https://pris.ly/d/mongodb-replica-set"
 )]
 pub struct MongoReplicaSetRequired {}
+
+#[derive(Debug, UserFacingError, Serialize)]
+#[user_facing(
+    code = "P2032",
+    message = "Error converting field \"{field}\" of expected type \"{expected_type}\", found incorrect value of \"{found}\"."
+)]
+
+pub struct MissingFieldsInModel {
```

Comment:
The error struct name suggest that you wanted to scope it to nulls when non-null was expected, but the error doesn't reflect that. I do like the more general message though, however: There's a subtle problem with the wording, because the type of the field is not indicative of nullability, it's the property of the field that does that. I could imagine that it's more useful as a user to have `Error converting field \"{field}\" of expected type \"{expected_type}\" (nullable: <true | false>), found incompatible value of \"{found}\`.